vhdbuilder/packer/install-dependencies.sh

```diff
@@ -115,6 +115,7 @@ if [[ $OS == $MARINER_OS_NAME ]]; then
     addMarinerNvidiaRepo
     overrideNetworkConfig || exit 1
     if grep -q "kata" <<< "$FEATURE_FLAGS"; then
+      installKataDeps
```

Comment:
this check in addition to the v1 check inside the func seems unnecessary? isn't kata v2 only anyway?

Reply:
Yes, for now it's 2.0 only. This is mostly for supporting future major version releases (like Kata v3).